python/graphstorm/config/argument.py

```diff
@@ -824,26 +824,26 @@ def use_early_stop(self):
 
     ## RGCN only ##
     @property
-    def n_bases(self):
+    def num_bases(self):
         """ Number of bases used in RGCN weight
         """
         # pylint: disable=no-member
-        if hasattr(self, "_n_bases"):
-            assert isinstance(self._n_bases, int)
-            assert self._n_bases > 0 or self._n_bases == -1
-            return self._n_bases
-        # By default do not use n_bases
+        if hasattr(self, "_num_bases"):
+            assert isinstance(self._num_bases, int)
+            assert self._num_bases > 0 or self._num_bases == -1
+            return self._num_bases
+        # By default do not use num_bases
         return -1
 
     ## RGAT only ##
     @property
-    def n_heads(self):
+    def num_heads(self):
         """ Number of attention heads
         """
         # pylint: disable=no-member
-        if hasattr(self, "_n_heads"):
-            assert self._n_heads > 0
-            return self._n_heads
+        if hasattr(self, "_num_heads"):
+            assert self._num_heads > 0
```

Comment:
Could you help to add the error message of this and the next line assertion? Thanks.